packages/core-transaction-pool/src/memory.ts

```diff
@@ -76,70 +91,77 @@ export class Memory {
         return this.byId[id].transaction;
     }
 
-    public getByType(type: number): Set<MemoryTransaction> {
-        const MemoryTransactions: Set<MemoryTransaction> = this.byType[type];
+    public getByType(type: number): Set<SequentialTransaction> {
+        const SequentialTransactions: Set<SequentialTransaction> = this.byType[type];
 
-        if (MemoryTransactions !== undefined) {
-            return MemoryTransactions;
+        if (SequentialTransactions !== undefined) {
+            return SequentialTransactions;
         }
 
         return new Set();
     }
 
-    public getBySender(senderPublicKey: string): Set<MemoryTransaction> {
-        const MemoryTransactions: Set<MemoryTransaction> = this.bySender[senderPublicKey];
+    public getBySender(senderPublicKey: string): Set<SequentialTransaction> {
+        const SequentialTransactions: Set<SequentialTransaction> = this.bySender[senderPublicKey];
 
-        if (MemoryTransactions !== undefined) {
-            return MemoryTransactions;
+        if (SequentialTransactions !== undefined) {
+            return SequentialTransactions;
         }
 
         return new Set();
     }
 
-    public remember(MemoryTransaction: MemoryTransaction, maxTransactionAge: number, databaseReady?: boolean): void {
-        const transaction: Interfaces.ITransaction = MemoryTransaction.transaction;
+    public remember(SequentialTransaction: SequentialTransaction, maxTransactionAge: number, databaseReady?: boolean): void {
```

Comment:
```suggestion
    public remember(sequentialTransaction: SequentialTransaction, maxTransactionAge: number, databaseReady?: boolean): void {
```

Variable names should always be in `camelCase`.

Reply:
You named it `MemoryTransaction` all over the place, I did not wanted to argue and assumed you wanted it to be with capital letter for some reason.

Reply:
I completely removed the `[sS]equentialTransaction` from the code in a followup PR: https://github.com/ArkEcosystem/core/pull/2494